api/DOMMatrixReadOnly.json

```diff
@@ -1655,7 +1655,9 @@
             "safari": {
               "version_added": "11"
```

Comment:
Does the collector agree that it's supported in Safari 11? https://mdn-bcd-collector.gooborg.com/tests/api/DOMMatrixReadOnly/transform tests false in STP, so was it removed? If so, was it added and removed in iOS too?

It doesn't seem very likely there are differences between desktop and mobile for this API.

Reply:
It seems like there's some sort of collector bug -- I'm getting a warning from `update-bcd` that doesn't make much sense: "warn: api.DOMMatrixReadOnly.transform skipped for safari because results are older than BCD".  Safari 16.1 doesn't support the feature either, so I'm going to close this PR and look into it some more.